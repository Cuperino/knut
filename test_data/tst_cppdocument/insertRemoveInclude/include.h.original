#ifndef INCLUDE_H
#define INCLUDE_H

#include "constants.h"

class Foo {

};

#endif // INCLUDE_H
