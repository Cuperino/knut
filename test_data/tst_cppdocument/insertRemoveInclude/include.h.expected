#ifndef INCLUDE_H
#define INCLUDE_H

#include "constants.h"

#include <bar.h>

class Foo {

};

#endif // INCLUDE_H
